src/components/Firealerts/api/internal/useFirealertsTriggers.tsx

```diff
@@ -0,0 +1,19 @@
+// import useSwr from 'swr';
+
+import useSWR from 'swr';
+import { useEmulatorConfig } from '../../../common/EmulatorConfigProvider';
+import { FirealertsTrigger } from '../../models';
+
+export function useFirealertsTriggers(): FirealertsTrigger[] { 
+    const config = useEmulatorConfig('eventarc');
+    const FIREALERTS_EVENT_TYPE = "google.firebase.firebasealerts.alerts.v1.published-google";
+    const fetcher = async (url: URL) => {
+        console.log(`Fetching FireAlerts triggers from ${url}`);
+        const response = await fetch(url);
+        const json = await response.json();
```

Comment:
This will also parse 4xx and 5xx error responses as JSON. Are we sure about this?